kuma/wiki/templates/wiki/document.html

```diff
@@ -429,4 +429,35 @@ <h2 class="offscreen">{{ _('Document Tags and Contributors') }}</h2>
     {{ js('helpfulness', async=True) }}
   {% endif %}
 
+  {% if waffle.flag('section_edit') %}
+    <script type="text/javascript">
+        (function($) {
+            $('#wikiArticle').children('h2[id]').each(function() {
+                var href = '{{ document.get_edit_url() }}#' + $(this).attr('id');
+
+                $('<a>').attr({
+                    href: href,
+                    'class': 'button section-edit only-icon'
+                })
+                .append($('<i aria-hidden="true" class="icon-pencil"></i>'))
+                .append($('<span>'))
+                .find('span')
+                .text(gettext('Edit'))
+                .end()
+                .on('click', function(e) {
+                    e.preventDefault();
+
+                    mdn.analytics.trackEvent({
+                        category: 'Section Edit',
+                        action: href
```

Comment:
Could we change this to `action: $(this).attr('id')`? That way in GA we'll see actions like "#Browser compatibility", "#Specifications", "#Examples" so we can learn which sections of pages are most popular for edits/contributions.